### PR TITLE
Organized readme.md of data structures and added time complexity

### PR DESCRIPTION
**Pull request by AnjumanHasan**
All the information in the same readme.md file makes it difficult to read and lengthy as well.
I created a folder called blobaccess which will work in helping to guide from one readme to another hence make the project look clean.
I have also added time complexities table in array and linked list

### CODE REVIEW

File: Data Structures/readme.md [hunk 1]
```diff
@@ -2,76 +2,16 @@
 In computer science, a data structure is a data organization, management, and storage format that enables efficient access and modification. More precisely, a data structure is a collection of data values, the relationships among them, and the functions or operations that can be applied to the data.
 
 ## Types of Data Structures
-- [Array](#array)
-- [Linked List](#linkedlist)
+- [Array](https://github.com/AnjumanHasan/computer_science/blob/main/BlobAccess/Array.md)
```
Comment (shhossain) [inline Data Structures/readme.md:5]:
```suggestion
- [Array](/BlobAccess/Array.md)
```
Change this

File: Data Structures/readme.md [hunk 2]
```diff
@@ -2,76 +2,16 @@
...
+- [Linked List](https://github.com/AnjumanHasan/computer_science/blob/main/BlobAccess/LinkedList.md)
```
Comment (shhossain) [inline Data Structures/readme.md:6]:
```suggestion
- [Linked List](BlobAccess/LinkedList.md)
```

And this

**Review by shhossain — CHANGES_REQUESTED** [review]

**Review by shhossain — CHANGES_REQUESTED** [review]
Each data structure should have a short description in Data Structure/readme.md file so you can just add time complexity to each data structure should do the job. And long descriptions like #30 in this PR should be added as extra.

**Comment by AnjumanHasan** [comment]
Applied the requested changes.
Should I open a different PR for that?

**Comment by shhossain** [comment]
> Applied the requested changes. Should I open a different PR for that?

No. Merging.

**Review by shhossain — APPROVED** [review]
No problem.

**Comment by AnjumanHasan** [comment]
I will be adding extra content in array and linked list files for visualisation of data structures.
Is that okay?

**Comment by shhossain** [comment]
Sure why not. @AnjumanHasan

**Pull request merged**